tests/flakes/follow-paths.sh

```diff
@@ -146,5 +146,5 @@ EOF
 
 git -C $flakeFollowsA add flake.nix
 
-nix flake lock $flakeFollowsA 2>&1 | grep "warning: input 'B' has an override for a non-existent input 'invalid'"
+nix flake lock $flakeFollowsA 2>&1 | grep "warning: input 'B' has a follows for a non-existent input 'invalid'"
```

Comment:
Not sure if it's helpful to have this distinction or not - can revert if not

Reply:
I'm not sure if we should make this distinction but if we do want to keep it this way I suggest changing the message to `input 'B' follows a non-existent input`

Reply:
Agreed!
```suggestion
nix flake lock $flakeFollowsA 2>&1 | grep "warning: input 'B' follows a non-existent input 'invalid'"
```

Reply:
Pushed